tests/cases/typecheck/property_access/out_of_bounds_access.stderr

```diff
@@ -1,6 +1,6 @@
-error[0016]: property `2` not found on type `(s517: i32, s518: i32)`
+error[0016]: property `2` not found on type `(s517: i32, s519: i32)`
```

Comment:
Unrelated to this PR, but perhaps we should change tuple type printing to omit name field if no name exists.